vscode/.vscodeignore

```diff
@@ -0,0 +1,11 @@
+*
+**
+
+!node_modules/got/dist/
```

Comment:
With this we don't bundle most of the dependencies we need, and at least on my machine I do not get hover or a code outline anymore. I am unsure whether we need to package all these files into the extension, but I believe most of them might be needed unless we change the build.